sdk/data/aztables/client.go

```diff
@@ -125,6 +125,10 @@ func createTableResponseFromGen(g *generated.TableClientCreateResponse) CreateTa
 // NOTE: creating a table with the same name as a table that's in the process of being deleted will return an *azcore.ResponseError
 // with error code TableBeingDeleted and status code http.StatusConflict.
 func (t *Client) CreateTable(ctx context.Context, options *CreateTableOptions) (CreateTableResponse, error) {
+	var err error
```

Comment:
I think we need to change the underlying line at 135, as well, to assign rather than shadow the 'err' declaration here.

Reply:
(I'm guessing this pattern needs to be applied in multiple locations)

Reply:
They're both in the same scope so there's no shadowing.

Reply:
I see, in this case the func() {} wrapper to defer means we use the latest declared 'err'. Ie, it's purposeful shadowing.

Reply:
To be pedantic, there is no shadowing here as all declarations of `err` are within the same scope. See the section [declarations and scope](https://go.dev/ref/spec#Declarations_and_scope) for the details.

Reply:
Pedantic is good! I see what you're saying now.